themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -7,6 +7,12 @@
 
   // More link should use default limit, not custom limit:
   $moreUrl = $this->url($params->getOptions()->getSearchAction()) . $results->getUrlQuery()->setPage(1)->setLimit($params->getOptions()->getDefaultLimit());
+
+  // Advanced search link
+  $advancedSearchAction = $params->getOptions()->getAdvancedSearchAction();
```

Comment:
To avoid unnecessary work, what if you change this line to:

```
$advancedSearchAction = ($currentSearch['advanced_link_bottom'] ?? false)
    ? $params->getOptions()->getAdvancedSearchAction() : false;
```

and then use `if ($advancedSearchUrl)` in the template logic below?

Reply:
On further reflection, I realize that this may not be ideal since it would make it more awkward to add advanced_link_top in the future... but I prefer to avoid generating URLs and doing options lookups if we know we're not going to use them. If you prefer to leave it as-is, I won't fight you, but maybe there's an intermediate solution that's a little more efficient. Let me know what you think! :-)

Reply:
I made the optimization -- I agree it's a small improvement and we can always undo it later if there's an advanced_link_top.